Monika-After-Story/MonikaModDev
# 2B spack

#9376 

# Key Changes
* adds 2b spack - thumbs is already included
* adds 2b o31 intro
* adds "hide-ring" ex-prop
## TODO
* [x] add o31 intro dialogue

# Testing
* verify looks good
* verify ring is hidden when outfit is worn and is worn again when outfit is taken off (if applicable)
* verify o31 intro event works
